canton-3x/shade_rule

```diff
@@ -0,0 +1 @@
+zap org.apache.pekko.stream.scaladsl.BroadcastHub*
```

Comment:
If possible I prefer to have final empty line.

Reply:
Done.